### ISSUE TITLE
Cannot Launch Studio to Run Code in Steps

### ISSUE THREAD

**Issue by phegel** [intersystems-community/RGateway#1]
Following the Steps at https://github.com/intersystems-community/RGateway/blob/master/README.md
I was able to get to step 3, but when I try to login to Studio to create a COS Routine(.mac),  I get access denied.

I'm able to run in terminal.

Do I need a license?

**Comment by eduard93** [comment]
Hello @phegel.
You do need a license.
Alternatively you can install RGateway into a new separate namespace on an existing server and R (with RServe) on a remote new server.
This way additional load on existing server would be minimal.

**Comment by phegel** [comment]
I recieved the license from Gayle on Friday and its now active on the system, but Studio still wouldn't let me login.  So I decided to compare my Security for my services.  

I edited the first one like this:

1. Navigated to System Administration > Security >Services
2. Click on the %Service_Bindings Name
3. Click Checkbox Servcie Enabled
4. Click Save

Did that with the very first one and it made Studio start working.

Thanks,
Paul
